specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-09-30/consumption.json

```diff
@@ -0,0 +1,4538 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-08-31",
```

Comment:
should this version be 2018-09-30?

Reply:
We want to upgrade all our versions at the same time to 9-30 when the UX is ready and hence the pause here.